api/SubmitJob/src/main.py

```diff
@@ -0,0 +1,27 @@
+import json
+from os import getenv
+
+import boto3
+
+client: boto3 = boto3.client('batch')
```

Comment:
I don't recognize this synax, I expected `batch = boto3.client('batch')`

Reply:
or `BATCH = boto3.client('batch')` if we're sticking with PEP8 rules for global names

Reply:
That was an artifact of me trying to do a type annotation, turns out boto3 builds all of it's classes based of a json spec, so it wasn't going to work anyway